wgpu-hal/src/gles/mod.rs

```diff
@@ -667,6 +667,7 @@ enum Command {
     ClearColorI(u32, [i32; 4]),
     ClearDepth(f32),
     ClearStencil(u32),
+    ClearDepthAndStencil(f32, u32),
```

Comment:
Could you just write a note why we need this?

Reply:
Done